### PR TITLE
DOCSP-38506 AtlasCLI 1.20.0 Release Notes

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
- [DOCSP-38506](https://jira.mongodb.org/browse/DOCSP-38506)
- [STAGING](https://preview-mongodbkanchanamongodb.gatsbyjs.io/atlas-cli/DOCSP-38506/atlas-cli-changelog/)
- [LATEST BUILD LOG](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=661d74853a5920a291681d1c)

### Self-Review Checklist

- [ ] Check that the submodule pulled in the right changes (if applicable).
- [ ] [Define](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) taxonomy [values](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) at top of page.
- [ ] Add genre facets (tutorial or reference), as in this [example PR](https://github.com/10gen/cloud-docs/pull/5042).
- [ ] Add programmingLanguage (if necessary).
- [ ] Add meta keywords (if necessary).
- [ ] Resolve any new warnings or errors in the build.
- [ ] Proofread for spelling and grammatical errors.
- [ ] Check staging for rendering issues.
- [ ] Confirm links are working.

### CODE REVIEW

File: snooty.toml [hunk 1]
```diff
@@ -29,7 +29,7 @@ aagent = "Automation Agent"
 adf = "Atlas Data Federation"
 atlas-cli = "Atlas CLI"
 atlas-cli-full = "MongoDB Atlas CLI"
-atlas-cli-version = "1.19.0"
+atlas-cli-version = "1.20.0"
```
Comment (sarahsimpers) [inline snooty.toml:32]: Please update the redirects file as well (Add a new version, Update the symlinks for the master and stable branches)

File: source/atlas-cli-changelog.txt [hunk 2]
```diff
@@ -14,6 +14,17 @@
...
    :class: singlecol
 
+.. _atlas-cli-1.20.0:
+
+{+atlas-cli+} 1.20.0
+--------------------
+
+Released 11 April 2024
+
+- Adds :ref:`atlas-projects-update` command.
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:24]: Nit
```suggestion
- Adds the :ref:`atlas-projects-update` command.
```

**Review by sarahsimpers — APPROVED** [review]
LGTM with two changes (one nit, one change)

**Review by tibulca — APPROVED** [review]
LGTM

**Pull request merged**